app/components/Comments/CommentView.tsx

```diff
@@ -27,23 +51,87 @@ const CommentView = (props: Props) => {
     contentTarget,
     style,
     displayTitle = true,
-    newOnTop = false,
     contentAuthors,
   } = props;
   const commentFormProps = {
     contentTarget,
   };
-  const tree = generateTreeStructure(comments);
+  const [ordering, setOrdering] = useState<Option>(orderingOptions[0]);
+  const [sortedComments, setSortedComments] = useState<Comment[]>(comments);
+  const [displaySorting, setDisplaySorting] = useState(false);
+  const tree = generateTreeStructure(sortedComments);
+
+  useEffect(() => {
+    const sorted = [...comments].sort((a: Comment, b: Comment) => {
+      if (ordering.value === 'createdAt') {
+        return moment(b.createdAt).valueOf() - moment(a.createdAt).valueOf();
+      } else if (ordering.value === 'createdAtInv') {
+        return moment(a.createdAt).valueOf() - moment(b.createdAt).valueOf();
+      } else if (ordering.value === 'reactionsGrouped') {
+        return a.reactionsGrouped.count - b.reactionsGrouped.count;
+      }
+      return 0;
+    });
+
+    setSortedComments(sorted);
+  }, [ordering, comments]);
+
   return (
     <div style={style}>
-      <Title displayTitle={displayTitle} />
+      <Flex
+        flex-row
+        justifyContent="flex-start"
+        alignItems="center"
+        className={styles.headerContainer}
+      >
+        <Title displayTitle={displayTitle} />
+        {
+          <Dropdown
+            show={displaySorting}
+            toggle={() => setDisplaySorting(!displaySorting)}
+            contentClassName={styles.changelogDropdown}
+            triggerComponent={
+              <Icon
+                name="arrow-down-wide-narrow"
+                size={20}
+                onClick={() => {
+                  setDisplaySorting(!displaySorting);
+                }}
+                className={styles.sortIcon}
+                iconNode={<ListFilter />}
+              />
+            }
+          >
+            <Dropdown.List>
+              {orderingOptions.map((o: Option) => {
+                return (
+                  <>
+                    <Dropdown.ListItem
+                      key={o.value}
+                      onClick={() => {
+                        setDisplaySorting(!displaySorting);
+                        setOrdering(o);
+                      }}
+                      className={styles.dropdownElement}
+                    >
+                      {o.label}
+                    </Dropdown.ListItem>
+                    <Dropdown.Divider />
+                  </>
+                );
+              })}
+            </Dropdown.List>
+          </Dropdown>
+        }
+      </Flex>
+
       <Flex column gap="var(--spacing-sm)">
         {!formDisabled && <CommentForm {...commentFormProps} />}
 
-        <LoadingIndicator loading={!comments}>
-          {comments && (
+        <LoadingIndicator loading={!sortedComments}>
+          {sortedComments && (
             <CommentTree
-              comments={newOnTop ? tree.reverse() : tree}
```

Comment:
Why is `newOnTop` removed? This is used in the bdb comments where the order is switched (newest first). I still think we need this, as it should indicate the "default" order of items

Reply:
Did you look into this? I think we should support default sorting so that it's newest first on bdb